DigitalExcellence/dex-frontend
# Feature/search feature on home and nav

## Description

Implemented the search functionality on the home page

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. On the home page enter a string in the search bar.
2. You should be redirected to the project overview page, and see the projects containing your search string.

## Link to issue

Closes: #166
